### PR TITLE
Added 'Getting Help' section

### PR DESCRIPTION
**Pull request by borjasotomayor**
I've added a new section called 'Getting Help' that includes two pages:

- **Asking Questions**: A generalized version of the question-asking guidelines we've provided in a number of classes.
- **Office Hours**: Tips/suggestions on how to get the most out of office hours.

We also need some sort of "getting help with this guide" page, but I'm not sure whether we want to put it in this section (which is supposed to be about how to get help in your classes), or under "About This Guide" (in PR #13). If the latter, calling the page "Getting Help" might be confusing, so we should think of a different name.

### CODE REVIEW

**Comment by ar0r** [comment]
I suggest we put information about how to get help with the guide in about and add a pointer to that section to this one.   I suggest "Getting Help in Your Classes" as a possible title.

File: getting-help/questions.rst [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  opposite is true in most CS classes: when asking for help, we prefer
+  that you *overwhelm* us with information.
+
+  In particular, it will be much easier for course staff to help you if they are
+  able to reproduce the exact issue you are encountering (i.e., when
+  they run your code, they must be able to observe the exact same issue
+  you’re encountering). And to do so, they need as much information as
+  possible from you:
+
+   -  If your question relates to your code, and your class uses GitHub
```
Comment (victor-almaraz) [inline getting-help/questions.rst]: I think the Github point applies to any class that uses version control. Perhaps we could generalize it.

Comment (victor-almaraz) [reply getting-help/questions.rst]: I also think we should change the title if we add the "getting help with this guide" section to the about page. Beyond that, I only have a small comment in the the asking questions page.

Comment (borjasotomayor) [reply getting-help/questions.rst]: @vaargueta: Anne just committed some changes to generalize the Git language. Does it look good to you? 

Also, my plan is to add a note saying something like this at the top of this page (after both this PR and the About PR are merged):

> This page provides some useful guidelines on how to ask questions in your classes (and more specifically on an online discussion board). If you have questions about the guide itself, please see the "Getting Help" page in the "About This Guide" section.

Do you think that would be enough?

Comment (victor-almaraz) [reply getting-help/questions.rst]: Yes, I think that addresses my concern. I think the note at the top of the page should be enough. If you want it to be particularly visible, I'd consider putting it in a note block just to draw the eye to it.

**Pull request merged**